diff --git a/src/util/build_config.h b/src/util/build_config.h
index 52c4f210..d6f1d788 100644
--- a/src/util/build_config.h
+++ b/src/util/build_config.h
@@ -135,7 +135,7 @@
 #define ARCH_CPU_ARMEL 1
 #define ARCH_CPU_32_BITS 1
 #define ARCH_CPU_LITTLE_ENDIAN 1
-#elif defined(__aarch64__)
+#elif defined(__aarch64__) || defined(_M_ARM64)
 #define ARCH_CPU_ARM_FAMILY 1
 #define ARCH_CPU_ARM64 1
 #define ARCH_CPU_64_BITS 1
